src/browser/focus.js

```diff
@@ -1,6 +1,12 @@
-import { sendKeys } from '@web/test-runner-commands';
+import { sendKeys, sendMouse } from '@web/test-runner-commands';
 
-export const focusWithKeyboard = async(element) => {
-	await sendKeys({ press: 'Escape' });
+export async function focusWithKeyboard(element) {
+	await sendKeys({ press: 'Shift' }); // Tab moves focus, Escape causes dismissible things to close
```

Comment:
As hinted at with the comment, when using this in core `Escape` was causing our dropdowns and dialogs to dismiss themselves. Core's copy uses `Tab` which was also problematic since it moves focus around and was why our [visual-diff copy](https://github.com/BrightspaceUI/visual-diff/blob/main/helpers/focus.js#L14) used `Escape`.

`Shift` _seems_ to be a safe option... we'll see though I guess.